sklearn/decomposition/kernel_pca.py

```diff
@@ -210,6 +211,11 @@ def _fit_transform(self, K):
                                                 maxiter=self.max_iter,
                                                 v0=v0)
 
+        # flip eigenvectors' sign to enforce deterministic output
+        # note: copying the second element is needed so that both inputs do
+        # not refer to the same object
+        self.alphas_, _ = svd_flip(self.alphas_, self.alphas_.copy().T)
```

Comment:
would it work to do 
```python
svd_flip(self.alphas_, np.empty_like(self.alphas_).T)
```
to avoid a memory copy?

Reply:
Correct since the value of the second argument does not affect our output. Done